src/NinjaTools.FluentMockServer.API/Proxy/Visitors/Collections/Converters/HeaderCollectionConverter.cs

```diff
@@ -0,0 +1,73 @@
+using System;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using YamlDotNet.Core;
+using YamlDotNet.Serialization;
+
+namespace NinjaTools.FluentMockServer.API.Proxy.Visitors.Collections.Converters
+{
+    public class HeaderCollectionConverter : JsonConverter<HeaderCollection>, IYamlTypeConverter
+    {
+        /// <inheritdoc />
+        public override void WriteJson(JsonWriter writer, HeaderCollection value, JsonSerializer serializer)
+        {
+     //       var t = JToken.FromObject((Dictionary<string, string[]>) value);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)